src/hooks/usePollQuery.ts

```diff
@@ -95,13 +98,17 @@ export const usePollQuery = <ApolloQueryVariables, ApolloQueryResultType>({
     search,
   ]);
 
+  // Responsible for clearing the refresh interval if the browser is offline
+  useEffect(() => {
+    if (isOffline) {
```

Comment:
we can restart polling when connectivity is restored with this function call.
`      pollQuery({
        search,
        resourceId,
        refetch,
        setIntervalId,
        getQueryVariables,
      });`
we may have to store the previous value of isOffline since the dependency array will have other items